app/controllers/idv/document_capture_controller.rb

```diff
@@ -71,6 +70,16 @@ def confirm_document_capture_needed
       redirect_to idv_ssn_url
     end
 
+    # This is copied from DocumentCaptureConcern, with out the step check
+    def override_document_capture_step_csp
```

Comment:
Can we rename this to something like `override_content_security_policy_to_allow_acuant`